README.md

```diff
@@ -32,3 +32,5 @@ without forcing Vault users to do a full Vault upgrade.
     an existing Kubernetes cluster.
 
   * A Docker image [`hashicorp/vault-k8s`](https://hub.docker.com/r/hashicorp/vault-k8s) is available. This can be used to manually run `vault-k8s` within a scheduled environment.
+
+  * Raw binaries are available in the [HashiCorp releases directory](https://releases.hashicorp.com/vault-k8s/). These can be used to run consul-k8s directly or build custom packages.
```

Comment:
Should this be vault-k8s?
```suggestion
  * Raw binaries are available in the [HashiCorp releases directory](https://releases.hashicorp.com/vault-k8s/). These can be used to run vault-k8s directly or build custom packages.
```

Reply:
+1, I think having this in both repos could be beneficial. Want to make sure it gets to https://github.com/hashicorp/vault-k8s so end users are aware they can download it from there, but having it here could also be useful for internal users/ those new to the project.

Reply:
> Should this be vault-k8s?

Thanks, fixed typo!

Reply:
> +1, I think having this in both repos could be beneficial. Want to make sure it gets to https://github.com/hashicorp/vault-k8s#installation so end users are aware they can download it from there, but having it here could also be useful for internal users/ those new to the project.

Thanks @mdeggies! I have moved the release dockerfile back to this repo. See comment to Jason above ^